src/components/Coaching/CoachingRow/CoachingRow.tsx

```diff
@@ -61,31 +66,50 @@ export const CoachingRow: React.FC<Props> = ({
     },
   });
 
+  const usersList = users.nodes
+    .map((user) => user.firstName + ' ' + user.lastName)
+    .join(', ');
+
   return (
     <>
       <CoachingRowWrapper role="listitem">
-        <CoachingNameText variant="h6" color="primary">
-          <NextLink
-            href={{
-              pathname: '/accountLists/[accountListId]/coaching/[coachingId]',
-              query: { accountListId: accountListId, coachingId: id },
-            }}
-            passHref
-          >
-            <Link flex={1} underline="hover">
-              {name}
-            </Link>
-          </NextLink>
-          <Button
-            onClick={(event) => {
-              event.preventDefault();
-              setConfirmingDelete(true);
-            }}
-            aria-label={t('Remove Access')}
-          >
-            <VisibilityOff />
-          </Button>
-        </CoachingNameText>
+        <Box display="flex">
+          <Box flex={1}>
+            <CoachingNameText variant="h6" color="primary">
+              <NextLink
+                href={{
+                  pathname:
+                    '/accountLists/[accountListId]/coaching/[coachingId]',
+                  query: { accountListId: accountListId, coachingId: id },
+                }}
+                passHref
+              >
+                <Link underline="hover">{name}</Link>
+              </NextLink>
+            </CoachingNameText>
+            <Typography variant="subtitle1">{usersList}</Typography>
+          </Box>
+          <Box>
+            <Typography
+              variant="h6"
+              sx={{ float: 'left', marginInline: '5px' }}
+            >
+              {t('Balance:')}{' '}
+              {balance && currencyFormat(balance, currency, locale)}
```

Comment:
Do we want to hide the balance if it is zero? It seems like we would still want to show it. Also, this will just render 0 instead of formatting it as currency if the balance is 0.

Reply:
I was thinking we should still show Balance: 0 if the balance is 0. But I guess we could show USD 0.

Reply:
Yeah, I think we should show the currency like for other amounts. But feel free to disagree.